### ISSUE TITLE
Date picker current date incorrect if page was not refreshed in browser today

### ISSUE THREAD

**Issue by filcuk** [go-vikunja/vikunja#251]
### Description

To reproduce:
1. Load Vikunja on day 1
2. Return to the page on day 2
3. You can navigate around, switch projects, complete tasks
4. When you open a date picker (for example setting a due date), day 1 will be defaulted to and marked as 'current' date

### Vikunja Version

v0.23.0

### Can you reproduce the bug on the Vikunja demo site?

Yes

**Comment by kolaente** [comment]
I'm unable to reproduce this. Can you clarify the third step a little more?

**Comment by filcuk** [comment]
Of course; what I meant by the 3rd step is that user can continue using Vikunja, but so long as the page isn't reloaded in browser, the 'today' date will not update.
This can cause one to set incorrect due dates, if not careful.

To reproduce, you must not close browser or reload page.
This, in my use case, is common by the tab being pinned (stays loaded) and the device hibernating between sessions (browser stays open for days).

This is on the latest Firefox. I would not expect it to be different elsewhere, but if this is unexpected behaviour, please let me know and I'll test further.

**Comment by kolaente** [comment]
Managed to reproduce it, but not reliably. Will look into a fix.

**Comment by kolaente** [comment]
I've looked into it and it seems like the problem is caused by flatpickr. Since the library is essentially unmaintained since two years, I'll migrate away from it in the short term, though not in time for the next release. Until then, this issue will stay open, just so that you know.

**Comment by kolaente** [comment]
Closing this as we'll move away from flatpickr.
